src/App.tsx

```diff
@@ -1,14 +1,45 @@
+import 'react-toastify/dist/ReactToastify.css';
+
+import {
+  Footer,
+  Navbar,
+  Sidebar,
+} from 'components';
 // @ts-ignore
 import Mockman from 'mockman-js';
+import { Home } from 'pages';
 import {
   Route,
   Routes,
 } from 'react-router-dom';
+import { ToastContainer } from 'react-toastify';
 
 export default function App() {
   return (
-    <Routes>
-      <Route path="/q" element={<Mockman />} />
-    </Routes>
+    <>
+      <header className="">
```

Comment:
You can remove the className attribute wherever it's not required.

Reply:
Hey, @tejas-muthya37  Thanks for pointing it out. I mentioned this is just the dummy code I left them as a placeholder. will fill them up with some meaningful full classes in the upcoming changes.